pkgs/by-name/ni/nixos-render-docs/src/nixos_render_docs/redirects.py

```diff
@@ -47,23 +47,49 @@ def __str__(self):
 
     If you moved content, add its new location as the first element of the redirects mapping.
     Please update doc/redirects.json or nixos/doc/manual/redirects.json!
-""") # TODO: automatically detect if you just missed adding a new location, and make a tool to do that for you
+""")
         if self.identifiers_without_redirects:
             error_messages.append(f"""
 Identifiers present in the source must have a mapping in the redirects file.
     - {"\n    - ".join(self.identifiers_without_redirects)}
 
     This can happen when an identifier was added or renamed.
-    Please update doc/redirects.json or nixos/doc/manual/redirects.json!
-""") # TODO: add tooling in the development shell to do that automatically and point to that command
+
+    Added new content?
+        redirects add-content <identifier> <path>
+
+    Moved existing content to a different output path?
+        redirects move-content <identifier> <path>
+
+    Renamed existing identifiers?
+        redirects rename-identifier <old-identifier> <new-identifier>
+
+    Removed content? Redirect to alternatives or relevant release notes.
+        redirects remove-and-redirect <identifier> <target-identifier>
+
+    Note that you need to run `nix-shell doc` or `nix-shell nixos/doc/manual` to be able to run this command.
```

Comment:
We could be slightly more clever (and concise) here by injecting this as a parameter as well, but this is not a blocker.

Reply:
On what command to run? I tried that before before when we were just pointing to the file paths, the path that the NRD receives is a nix store path, unless we trim it down it'll be quite verbose

Reply:
Just as you pass `--file` you may pass something like `--root` and output that in the error message.